tests/hotplug/cpu.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"strconv"
```

Comment:
Commit message: cpu hotplug: test actual guest memory

CPU count?

Reply:
Thanks! It was obviously a leftover :)
done